src/list-data/entity-type-data.ts

```diff
@@ -296,14 +299,8 @@ export const EntityTypesXproData: EntityI[] = [
       'Reinstate an extraprovincially registered company that is no longer active with BC Registries.'
     ],
     amlBlurbs: [
-      [
-        `Register an amalgamation that occurred in another jurisdiction where at least one of the companies is
-           extraprovincially registered in BC.`
-      ],
-      [
-        `Register an amalgamation or merger that occurred in another jurisdiction where at least one of the companies
-          is extraprovincially registered in BC.`
-      ]
```

Comment:
This used to be an array to distinguish between Canadian and International xpro amalgamations. Since the dialog tooltip is displayed before the home jurisdiction is selected, I combined these blurbs.

Now looks like this:
![image](https://github.com/bcgov/namerequest/assets/10002889/e36f17d7-61e1-4ba7-b0b1-ea831144a3c0)